### PR TITLE
Fixes #1305

### LINKED ISSUE #1305: Autocomplete install does not work in zsh
| Q                 | A
| ----------------- | ---
| Issue Type        | Bug
| Deployer Version  | 5.1.3
| Local Machine OS  | Ubuntu 14.04.5 LTS
| Remote Machine OS | N/A

### Description 
autocomplete install does not work. I guess I am missing some package, but maybe script should give me some clue what I am missing?

### Steps to reproduce
$ dep autocomplete --install > ~/.deployer_completion && echo "source ~/.deployer_completion" >> ~/.zshrc
$ source ~/.zshrc

### Output log
/home/vytautas/.deployer_completion:47: command not found: complete

### PR DESCRIPTION
**Pull request by Schrank**
@campaignupgrade works this for you?

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Deployer comes with autocomplete support for task names, options, and hosts.
 Add the following to your `~/.bashrc` or `~/.zshrc`:
 
 ```
-eval "$(dep autocomplete --shell bash)"
+eval "$(dep autocomplete --install | sudo tee /etc/bash_completion.d/deployer)
```
Comment (antonmedv) [inline docs/installation.md:19]: Let’s drop sudo tee file part

**Comment by campaignupgrade** [comment]
`dep autocomplete --install` seems to only echo the script back to stdout. It doesn't appear to execute the script. 

I can't say I really understand what it is trying to do, but do you want `dep autocomplete --install | sudo tee /etc/bash_completion.d/deployer` instead?

**Comment by antonmedv** [comment]
There is also `eval`.

**Comment by campaignupgrade** [comment]
What should I look for to test if this is working? 

I understand it is about autocomplete, but what kinds of things should it be autocompleting?

**Comment by antonmedv** [comment]
Tasks names and cli options.

**Pull request merged**